### ISSUE TITLE
Tool Tip needed for the Accessibility Tool bar buttons

### ISSUE THREAD

**Issue by HerinHentry** [mr-bobz/talarts#159]

**Comment by mr-bobz** [comment]
I see that the buttons have this - <span class="offscreen">Toggle High Contrast</span>. So the screen reader should be reading it, why do we need a Tool Tip then?

**Comment by HerinHentry** [comment]
For a sighted user, who is not very teechnical / not used to these icons, it's hard to understand.
I had the site reviewed by few people and this was one of the feedback. Especially colour contrast mode is only for sighted users. Hence this should be supported.

**Comment by mr-bobz** [comment]
Added title via jQuery!

**Comment by HerinHentry** [comment]
Excellent! Works now. Thank you.
